package.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
Isn't necessary to declare a root package json file?

Reply:
I was thinking that we don't really need to bother here; the internal packages don't need to be linked to each other and I'd treat them as standalone code repos just centralized for organizational purposes.